### DISCUSSION TITLE
Is Android supported?

### DISCUSSION THREAD

**Discussion by cowwoc** [Authenticator-Extension/Authenticator#1153] [Q&A]
https://github.com/Authenticator-Extension/Authenticator lists Chrome, Firefox, Windows Store and iOS App Store. Android Play Store is nowhere to be found. Does this mean that Android is not supported?

The goal here is to have a Windows and Android installation sync with each other automatically. Is this possible?

**Comment by eadmaster** [comment]
official chrome app on android does not support extensions, only [Firefox beta does](https://github.com/Authenticator-Extension/Authenticator/discussions/1142)
